payment.yaml

```diff
@@ -0,0 +1,187 @@
+    parameters:
+      - $ref: '#/components/parameters/RequestId'
+      - $ref: '#/components/parameters/BlockingTime'
+      - in: path
+        name: transaction_id
+        schema:
+          type: string
+        required: true
+        description: blockchain transaction id
+    get:
+      tags:
+        - Transactions
+      description: get transaction info
+      operationId: getTransaction
+      responses:
+        '200':
+          description: Successfully returned transaction info
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Transaction'
+        '404':
+          description: Transaction not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        default:
+          description: Generic Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+  /wallets/{public_address}:
+    parameters:
+      - $ref: '#/components/parameters/RequestId'
+      - $ref: '#/components/parameters/BlockingTime'
+      - in: path
+        name: public_address
+        schema:
+          type: string
+        required: true
+        description: public address of wallet to create/ fund in native currency
+    post:
+      tags:
+        - Transactions
+
+      description: get transaction info
+      operationId: createWallet
+      responses:
+        '204':
+          description: Successfully created wallet
+        default:
+          description: Generic Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  
+  /wallets/{public_address}/transactions:
+    parameters:
+      - $ref: '#/components/parameters/RequestId'
+      - $ref: '#/components/parameters/BlockingTime'
+      - in: path
+        name: public_address
+        schema:
+          type: string
+        required: true
+        description: public address of wallet
+    post:
+      tags:
+        - Transactions
+      description: pay a user some KIN
+      operationId: payTo
+      requestBody:
+        description: amount
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                amount:
+                  type: integer  # in microKIN
+                meta:
+                  type: string
+                  example: "order:1234"
+      responses:
+        '200':
+          description: Successfully returned transaction info
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Transaction'
+        '204':
+          description: blocking time was timed out before the transaction was fully submitted. We return the transaction_id, so it can be polled on later.
```

Comment:
we need to discuss it. Horizon timeout is 60 sec, not sure this case is applicable

Reply:
I removed the blocking time -> all post operations are fire and forget. get requests only return data that they have (no waiting)